### PR TITLE
Added header and footer in the Map UI theme

### PR DESCRIPTION
**Pull request by maeisabelle**
Kindly review the changes made. Refer to the commit message for reference. Thanks!

### CODE REVIEW

**Comment by grtjn** [comment]
Looks okay to me, but I'd like @jenbreese to have a look at the css as well..

**Comment by maeisabelle** [comment]
Hi @grtjn and @jenbreese,

I also added changes on the detail controller so the marker for the selected documented is the only one shown when a document is clicked from the search results (same goes with the one in the Similar section). It also changes the center of the map to the position of the marker. Kindly review.

**Comment by grtjn** [comment]
Looks good to me. I might have some more changes for mapManager that might interfere with this, but I'll merge that locally before pushing, and only after this PR is done..

**Comment by grtjn** [comment]
@jenbreese Any comments on css code and styling from your side?

File: app/templates/ui/styles/main.less [hunk 1]
```diff
@@ -124,6 +124,7 @@ nav.nav {
 
 /*footer*/
 footer {
+  position: fixed;
```
Comment (jenbreese) [inline app/templates/ui/styles/main.less]: This is the main.less and the change needs to be in the map/theme.less

**Comment by grtjn** [comment]
Merging this to make testing easier for Jen.. (and I might wanna pile up something as well..)

**Comment by jenbreese** [comment]
@maeisabelle  and @grtjn I was able to test this and it worked fine. However, I didn't see the actual map controls. Are those going to be added later?

**Pull request merged**